planck-cljs/src/planck/io.cljs

```diff
@@ -346,12 +346,67 @@
 (defn ^boolean directory?
   "Checks if dir is a directory."
   [dir]
-  (js/PLANCK_IS_DIRECTORY (:path (as-file dir))))
+  (boolean (js/PLANCK_IS_DIRECTORY (:path (as-file dir)))))
 
 (s/fdef directory?
   :args (s/cat :dir (s/or :string string? :file file?))
   :ret boolean?)
 
+(defn exists?
```

Comment:
Will need a `:refer-clojure` `:exclude` for `exists?` to avoid
```
WARNING: exists? already refers to: cljs.core/exists? being replaced by: planck.io/exists? at line 355 src/planck/io.cljs
```

Reply:
Fixed in rev 3055322.